demos/appyx-interactions/ios/src/iosMain/kotlin/AppyxSample.kt

```diff
@@ -0,0 +1,135 @@
+import InteractionTarget.Element
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterHorizontally
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clipToBounds
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.bumble.appyx.interactions.core.AppyxComponent
+import com.bumble.appyx.interactions.core.model.BaseAppyxComponent
+import com.bumble.appyx.interactions.core.ui.helper.AppyxComponentSetup
+import com.bumble.appyx.interactions.core.ui.output.ElementUiModel
+import theme.color_primary
+import theme.md_amber_500
+import theme.md_blue_500
+import theme.md_blue_grey_500
+import theme.md_cyan_500
+import theme.md_grey_500
+import theme.md_indigo_500
+import theme.md_light_blue_500
+import theme.md_light_green_500
+import theme.md_lime_500
+import theme.md_pink_500
+import theme.md_teal_500
+import kotlin.random.Random
+
+@Suppress("UnstableCollections") // actions parameter
+@Composable
+internal fun AppyxSample(
```

Comment:
Can we have this in a shared place? Is there anything platform-specific about it?

Reply:
See https://github.com/bumble-tech/appyx/pull/601#discussion_r1335682325